### PR TITLE
Add note about extensions needing to be OSS

### PR DESCRIPTION
**Pull request by haydennyyy**
### Describe in detail the problem you had and how this PR fixes it
Some people might think this is a catch-all for every VS Code need in the browser, I'm just informing them that only the open source extensions can be used (i.e. no VS Live Share, etc.)

### Is there an open issue you can link to?
Nope.

### CODE REVIEW

**Comment by bpmct** [comment]
Will you still be able to upload closed-source extensions via a VSIX file (as soon as that gets working #92)? Or are there other problems associated with that?

**Comment by NGTmeaty** [comment]
Closed-source extensions with VSIX will work after #92 gets fixed. I'm going to make a slight change here.

File: README.md [hunk 1]
```diff
@@ -62,6 +62,10 @@ How to [secure your setup](/doc/security/ssl.md).
 - Electron and ChromeOS applications to bridge the gap between local<->remote.
 - Run VS Code unit tests against our builds to ensure features work as expected.
 
+### Notes
+
+- VS Code Marketplace cannot be used, we're not allowed to use it. However, the open source extensions should all be allowed to work under `code-server` just fine.
```
Comment (NGTmeaty) [inline README.md]: My recommendations for the change here would be:

`At the moment we can't use the official VSCode Marketplace. Coder has created a collection of extensions that are open-source. While you can't directly download extensions from the official VSCode Marketplace, you can upload them to be installed in `code-server`.`

**Comment by haydennyyy** [comment]
@NGTmeaty Done, I agree, it looks better

File: README.md [hunk 2]
```diff
@@ -62,6 +62,10 @@ How to [secure your setup](/doc/security/ssl.md).
 - Electron and ChromeOS applications to bridge the gap between local<->remote.
 - Run VS Code unit tests against our builds to ensure features work as expected.
 
+### Notes
+
+- At the moment we can't use the official VSCode Marketplace. Coder has created a collection of extensions that are open-source. While you can't directly download extensions from the official VSCode Marketplace, you can upload them to be installed in code-server.
```
Comment (kylecarbs) [inline README.md]: Might be better to phrase this as:
`We've created a custom extension marketplace focused around open-sourced extensions.`
I'd like to convey that we don't just package a few additional exts, and it is a full marketplace that will be improved.

**Comment by haydennyyy** [comment]
Alright. I've edited it, and kept in the little bit on the end about being able to if you have the VSIX file

**Comment by coderasher** [comment]
I'm thinking we might want to wait until https://github.com/codercom/code-server/issues/92 is resolved since installing extensions manually won't work until that issue is resolved.

**Comment by NGTmeaty** [comment]
VSIX has a active PR 🎉 #245

**Comment by kylecarbs** [comment]
Will merge once https://github.com/codercom/code-server/pull/245 is in!

**Comment by NGTmeaty** [comment]
#245 is in. Merge?

**Pull request merged**